addons/binding/org.openhab.binding.rfxcom/src/main/java/org/openhab/binding/rfxcom/handler/RFXComBridgeHandler.java

```diff
@@ -62,12 +65,52 @@
 
     private List<DeviceMessageListener> deviceStatusListeners = new CopyOnWriteArrayList<>();
 
-    private static byte seqNbr = 0;
-    private static RFXComTransmitterMessage responseMessage = null;
-    private Object notifierObject = new Object();
     private RFXComBridgeConfiguration configuration = null;
     private ScheduledFuture<?> connectorTask;
 
+
+    private class TransmitQueue {
+        private Queue<RFXComBaseMessage> queue = new LinkedBlockingQueue<RFXComBaseMessage>();
+
+
+        public synchronized void enqueue(RFXComBaseMessage msg) throws IOException {
+            boolean wasEmpty = queue.isEmpty();
+            if (queue.offer(msg)) {
+                if (wasEmpty) {
+                    send();
+                }
+            } else {
+                logger.error("Transmit queue overflow. Lost message: {}", msg);
```

Comment:
> All that's happened is that we couldn't handle that one specific message. Nothing more.

So does this qualify as an error (which should be only be used for unrecoverable situations)?
Warn should be enough and if this is a regular phenomenon (and thus normal/expected), it might even be ok to decrease it to debug.